tests/geometry/test_point_types.py

```diff
@@ -63,6 +64,18 @@ def test_operations_mypy_cast():
         import mypy.api
     except ModuleNotFoundError:
         return
+    import sys
+
+    python_version = sys.version_info
+    if python_version[0] == 3 and (
+        python_version[1] > 10 or python_version[1] == 10 and python_version[2] > 6
```

Comment:
Nit
```
if python_version[0] >= 3 and python_version[1] >= 10 and python_version[2] >= 7
```

Reply:
I think this doesn't work because it won't catch something like 3.11.1 (fails the third check).

Reply:
Oh yeah, haha